modules/rosa-create-objects.adoc

```diff
@@ -546,4 +597,53 @@ $ rosa create ocm-role [flags]
 
 |===
 
-For more information about the ocm-roles and the roles created with the `rosa create ocm-roles`, see the Additional resources section.
\ No newline at end of file
+For more information about the OCM role created with the `rosa create ocm-role` command, see _Account-wide IAM role and policy reference_.
+
+[id="rosa-create-user-role_{context}"]
+== create user-role
+
+Create the needed user-role resources for your cluster.
```

Comment:
"needed" sounds a little funny to me. Again I see it was already there for other descriptions, but wondering if "necessary" or "required" would read better.

Reply:
Good suggestion, thanks. I have subsequently replaced the work with "required".